setup.cfg

```diff
@@ -16,19 +16,20 @@ classifiers =
     Intended Audience :: Science/Research
     License :: OSI Approved :: BSD License
     Programming Language :: Python
-    Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
 
 [options]
 zip_safe = False
 include_package_data = True
 packages = find:
-python_requires = >=3.6
+python_requires = >=3.7
```

Comment:
should probably be a separate PR which updates the state of things, bumps versions, etc

Reply:
This is the minimal requirement for `tornado_openapi3`. But if we make it an extra, indeed, this can go away.